homeassistant/core.py

```diff
@@ -439,7 +439,8 @@ def verify_event_loop_thread(self, what: str) -> None:
 
             # frame is a circular import, so we import it here
             frame.report(
-                f"calls {what} from a thread. "
+                f"calls {what} from a thread other than the event loop, "
+                "which is a non-thread-safe operation and may cause a crash. "
```

Comment:
I'd like to clarify why we would not have a deprecation period for incorrectly calling an async API.  Some developers didn't seem to understand the severity of doing a non-thread-safe operation, so I didn't use the word "instability" here, as I thought conveying that the system could crash was essential.

Reply:
I don't think we have to explain it's a non-thread-safe operation. Focus on the impact as that might cause people to actually care. We can even highlight other side effects:

```suggestion
                "which may cause Home Assistant to crash or data to corrupt. "
```